pkg/operator/quorumguardcontroller/quorumguardcontroller_test.go

```diff
@@ -64,14 +64,14 @@ controlPlane:
 			fields: fields{
 				client:   fakecore.NewSimpleClientset(deployment, &clusterConfigFullHA),
 				infraObj: haInfra},
-			expectedHATopology: configv1.HighlyAvailableTopologyMode, expectedEvents: 1, wantErr: false, expectedReplicaCount: 3,
+			expectedHATopology: configv1.HighlyAvailableTopologyMode, expectedEvents: 2, wantErr: false, expectedReplicaCount: 3,
```

Comment:
if deployment exists it will not contain pod affinity so an additional event will be required.